src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -17,7 +23,7 @@ public Navigate toPage(String pageName) {
     String host = UrlBuilder.getHostForWiki();
     String cacheBuster = pageName.equals("") || pageName.equals("/") ? "" : "?cb=" + DateTime.now().getMillis();
 
-    driver.get("http://" + host + pageName + cacheBuster);
+    driver.get("http://" + host + pageName + cacheBuster + "&" + defaultTestingGroup);
```

Comment:
Did you think about using [appendQueryStringToURL](tests/blob/master/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L71) method for appending URL params to URL?